screeps-start.js

```diff
@@ -145,6 +223,13 @@ const start = async () => {
   installPackages();
   writeModsConfiguration();
 
+  const updateOpt = process.argv.includes("--update");
+  const updateNeeded = updatePackages(updateOpt || config.autoUpdate);
+
+  if (updateOpt) {
+    process.exit(updateNeeded ? 1 : 0);
+  }
```

Comment:
I wanted to make it so that running `start --update` would return 0 if nothing needed an update and 1 otherwise. But yeah, I realize it's a bit silly; since you passed the flag, then the update will happen.